### PR TITLE
Added instructions on blog creation

### PR DESCRIPTION
**Pull request by kathatherine**
Added some info on blog posts to the contribution guide. I know there's more metadata we'll support and we'll want to iterate on the presentation of my instructions as well, so I thought I'd get it out there.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -190,15 +190,40 @@ that can help you.
...
 
-### Creating a blog post
+conda.org blog posts support the following metadata:
 
-*Coming soon! We are still figuring out exactly how this will look*
+```
+title: The title of your blog.
+slug: The name of your blog page within the conda.org url.
+tags: Tags to group your blog with others and enable better filtering and browsing (e.g. conda, release, package, announcement).
+tease: (Optional) A short summary of or exceprt from your blog. If this is not supplied, conda.org will use the first 20 words of your blog post.
```
Comment (kathatherine) [inline CONTRIBUTING.md:205]: I can't remember if "20 words" is correct, or if it's a certain number of characters.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -190,15 +190,40 @@ that can help you.
 
 ## Content contributors
 
-### Creating a news post
+### Creating a blog post
 
-*Coming soon! We are still figuring out exactly how this will look*
+Blog posts must be created in the ``conda-dot-org/blog`` directory with a file naming convention of ``YYYY-MM-DD-title.md``. 
 
+Let's say you're announcing a cool new project you've recently released. Your file would be named something like ``2023-04-22-cool-project-v1-release.md``.
```
Comment (kathatherine) [inline CONTRIBUTING.md:197]: We can also have them put the blog date in the metadata, but I figured having it in the file name would be a good way to organize the blog files themselves and Docusaurus can strip out and use the dates if presented this way in the file name, to my knowledge.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -190,15 +190,40 @@ that can help you.
...
 
-### Creating a blog post
+conda.org blog posts support the following metadata:
 
-*Coming soon! We are still figuring out exactly how this will look*
+```
+title: The title of your blog.
```
Comment (kathatherine) [inline CONTRIBUTING.md:202]: What other metadata do we support? Authors? Description? Other stuff?

**Review by kathatherine — COMMENTED** [review]
Added comments to my current commit

File: CONTRIBUTING.md (hunk 2 above)
Comment (travishathaway) [reply CONTRIBUTING.md:197]: That's a good idea.

File: CONTRIBUTING.md (hunk 3 above)
Comment (travishathaway) [reply CONTRIBUTING.md:202]: @kathatherine , take a look at the Docusaurus docs for this:

- https://docusaurus.io/docs/blog#adding-posts
- https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog#markdown-front-matter

File: CONTRIBUTING.md (hunk 1 above)
Comment (travishathaway) [reply CONTRIBUTING.md:205]: 20 words. 20 characters would be barely be a couple of words! 😅

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -190,15 +190,40 @@ that can help you.
...
+slug: The name of your blog page within the conda.org url.
+tags: Tags to group your blog with others and enable better filtering and browsing (e.g. conda, release, package, announcement).
+tease: (Optional) A short summary of or exceprt from your blog. If this is not supplied, conda.org will use the first 20 words of your blog post.
+```
+
+Consider the Cool Project example. Its file will begin with the following metadata, set off with three dashes, followed by the actual blog text.
```
Comment (travishathaway) [inline CONTRIBUTING.md:208]:
```suggestion
Consider the following "Cool Project" example. Its file will begin with the following metadata, set off with three dashes, followed by the actual blog text.
```

Quotes work there, right?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -190,15 +190,40 @@ that can help you.
 
 ## Content contributors
 
-### Creating a news post
+### Creating a blog post
 
-*Coming soon! We are still figuring out exactly how this will look*
+Blog posts must be created in the ``conda-dot-org/blog`` directory with a file naming convention of ``YYYY-MM-DD-title.md``.
```
Comment (travishathaway) [inline CONTRIBUTING.md:195]:
```suggestion
Blog posts must be created in the `blog` directory with a file naming convention of `YYYY-MM-DD-title.md`. 
```
@kathatherine, I think just saying `blog` directory would be okay. Reading `conda-dot-org/blog` confuses me a little as I'm likely to take this literally and assume there is actually a folder called `conda-dot-org` in this repository.

Also, in Markdown, you don't need to use double backticks. Just one will work.

**Review by travishathaway — CHANGES_REQUESTED** [review]
Thanks for this addition. Please see my suggestions for improvement.

**Comment by kathatherine** [comment]
@travishathaway Thanks for your patience on these changes. I believe I've fixed/added everything based on your feedback.

**Review by travishathaway — APPROVED** [review]

**Pull request merged**